lib/plop-pack/skel/component/README.md.hbs

```diff
@@ -42,20 +42,26 @@ pnpm install
 
 ### Testing
 
+Run the `jest` unit tests.
```

Comment:
I wonder if this should go beyond unit tests for some packages.

Then again, maybe functional testing is best done outside of NPM and on an artefact such as a Docker image?..

Reply:
I suppose it depends on the complexity of the item under test; some components do little more than a presentation render, for those this template is perfectly adequate.
What might be nice though, is an extra plop, that can add the interaction level test templates/assets/resources/config to bring up a simple running app to allow acceptance level testing of a component.
Presumably some of this config/resources could be useful to the generated apps as well :thinking:

Reply:
I think I missed that this was for the React components. Enzyme (or a replacement) should give us the interaction testing we need.

Reply:
I think we should drop `jest` though. The user probably doesn't care and who knows, maybe we will change the harness at some point.